src/NBitcoin/ConsensusFactory.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
```

Comment:
is this guy the new code in NBitcoin? or is it our code?

Reply:
I assume you mean the ConsensusFactory, this is from NBitcoin yes, its quite elegant in my opinion.

Reply:
I was asking because of the code style, so as long as it is from nbitcoin, no comment :)

Reply:
haha no I will decorate it today with comments